docs/testnet/tutorial.md

```diff
@@ -0,0 +1,158 @@
+
+
+This works. You now have an Obscuro contract that will let players play the guessing game. But it's not very good. So how do players know if they've won or not? We need events!
+
+In Ethereum, events are typically written to the event log that is viewable by everyone. In Obscuro, events are viewable only to those they're intended for. This is done by encrypting events with a key that is known only to the intended recipient. This is handled automatimatically for the recipient through the Wallet Extension.
+
+So let's go ahead and start adding in some events. First, we'll need to emit events letting a player know after they've played whether they have won or not.
+
+Events in Obscuro look exactly like events in any other EVM-based platform. All the complexity of ensuring privacy is abstracted away. So let's go ahead and define two new events in our contract.
+
+```
+event Correct(address indexed player, uint8 guess, uint prize);
+event Incorrect(address indexed player, uint8 guess, uint prize);
+```
+
+Where **prize** represents the current prize pool so we can remind the player of what they're playing for.
+
+With these events in place, we can now call them accordingly within our attempt function, so we end up with something like this:
+
+![Step 1 screenshot](https://images.tango.us/workflows/263b9f82-3d6e-46d1-908f-952591a206d8/steps/cb679525-fb5f-4cae-991a-02285beee941/765986b3-1999-4814-a6ac-89bb6d22488d.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&fp-z=1.0000&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=870%3A269)
+
+
+Before we move on, we've seen how we can keep things private (hint: all of the above is private!). What about if there are things we want to intentionally make public? Foe example, it makes sense for the current prize pool to be public, so let's go ahead and explore that. It's straightforward. All you do is create a public view that returns something - precisely the same as any other EVM contract!
+
+Our function to return the prize pool is:
+
+![Step 1 screenshot](https://images.tango.us/workflows/74ec7481-8c33-401f-8476-07c777e7cd3d/steps/d4a753b9-f507-4f6c-b080-40587ec82c00/8ffdc581-6e95-40bb-9c11-d3fa0ef01d84.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&fp-z=1.0000&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=518%3A87)
+
+And that's it! Congratulations on writing your first Obscuro smart contract!
+
+You can now go ahead and deploy the contract following the instructions [here](https://docs.obscu.ro/testnet/deploying-a-smart-contract.html)
```

Comment:
I think it would be good to have information here on how to interact with the smart contract - e.g. how to make guesses.